signac/contrib/project.py

```diff
@@ -85,15 +89,16 @@ def __setitem__(self, key, value):
 class Project:
     """The handle on a signac project.
 
-    Application developers should usually not need to
-    directly instantiate this class, but use
-    :meth:`~signac.get_project` instead.
+    A :class:`Project` may only be constructed in a directory that is already a
+    signac project, i.e. a directory in which :meth:`~signac.init_project` has
```

Comment:
This might need `:func:` instead of `:meth:`?
```suggestion
    signac project, i.e. a directory in which :func:`~signac.init_project` has
```